app/controllers/quote_anonymous_controller.rb

```diff
@@ -0,0 +1,18 @@
+# Entry point for the quotes_anonymous resource
+class QuoteAnonymousController < ApplicationController
```

Comment:
Omdat het gaat om een actie van quote misschien beter om dit te nesten 
```suggestion
class Quotes::AnonymizationController < ApplicationController
```